diesel/src/types/impls/date_and_time/mod.rs

```diff
@@ -67,12 +67,10 @@ impl PgInterval {
     }
 }
 
-primitive_impls! {
-    Date -> (PgDate, 1082, 1182),
-    Interval -> (PgInterval, 1186, 1187),
-    Time -> (PgTime, 1083, 1183),
-    Timestamp -> (PgTimestamp, 1114, 1115),
-}
+primitive_impls!(Date -> (PgDate, pg: (1082, 1182), sqlite: (Text)));
+primitive_impls!(Interval -> (PgInterval, pg: (1186, 1187)));
+primitive_impls!(Time -> (PgTime, pg: (1083, 1183), sqlite: (Text)));
+primitive_impls!(Timestamp -> (PgTimestamp, pg: (1114, 1115), sqlite: (Text)));
```

Comment:
These numbers are confusing to me. I appreciate there's some really specific implementation details, but it'd be good to expose what they are with english language. Can we make them less confusing?